manage.py

```diff
@@ -22,14 +22,14 @@ def coverage_context():
     cov.save()
     print(f'Coverage is {cov.report()}%')
 
+    cov.html_report()
+    cov.xml_report()
```

Comment:
How about printing out a line explaining that an xml report is being written to disk? What does it return, by the way? A path to `coverage.xml`? Perhaps you could do the same as in the f-string above?

Reply:
Alright, according to https://coverage.readthedocs.io/en/latest/api_coverage.html#coverage.Coverage.xml_report, the same float is returned.

Reply:
Thank you, it was changed.

Reply:
marking as resolved.